mongodbatlas/whitelist_api_keys.go

```diff
@@ -6,6 +6,13 @@ import (
 	"net/http"
 )
 
+/* IMPORTANT: DEPRECATION NOTICE
+Access List Replaces Whitelist
+Atlas now refers to programmatic API key whitelists as access lists.
+Atlas has deprecated the whitelist method and will disable it in June 2021.
+Please update any dependent work to use accesslist_api_keys.go
+*/
```

Comment:
I think if we follow this format https://github.com/golang/go/wiki/Deprecated then tools will pick up the fact that this is a deprecation announcement automatically

Reply:
Thank you so much @jamesbroadhead, did not know that!   Added commit to fit the spec.